crates/indexer/src/geyser/accounts/receipt.rs

```diff
@@ -275,6 +275,8 @@ pub(crate) async fn process_bid_receipt(
         client
             .dispatch_dialect_offer_event(key, bid_receipt.metadata)
             .await?;
+    } else {
+        trace!("Skipping Dialect dispatch for offer");
```

Comment:
Thinking this should be a debug log. Most offers as we reindex accounts will skip registering an offer event.